dev/ansible/proxy-minio.sh

```diff
@@ -0,0 +1,2 @@
+consul connect proxy -service minio-local -upstream minio:9999 -log-level debug </dev/null &>/dev/null & PROXY_PID_ID=$!
+echo $PROXY_PID_ID
```

Comment:
Isn't this inline in ansible?

Reply:
I left it, due that PR is still a draft. I will remove it, once PR is ready